src/hooks/useMe.ts

```diff
@@ -9,33 +8,23 @@ export interface Subject {
   subject: Patient;
 }
 
-interface Entry {
-  resource: Patient;
-}
-
-interface MeResponse {
-  resourceType: 'Bundle';
-  entry: Entry[];
-}
-
 export function useMe() {
-  const { httpClient } = useHttpClient();
+  return useRestQuery(
+    'GET /v1/fhir/dstu3/$me',
+    {},
+    {
+      select: (data) => {
+        const subjects: Subject[] = data.entry.map((entry) => ({
+          subjectId: entry.resource.id,
+          projectId: entry.resource.meta?.tag?.find(
+            (t) => t.system === 'http://lifeomic.com/fhir/dataset',
+          )?.code!,
```

Comment:
Is the code guaranteed for sure here?

Reply:
If you notice, it was already being force-casted before.